mrp_attachment_mgmt/README.rst

```diff
@@ -0,0 +1,106 @@
+===================
+Mrp Attachment Mgmt
+===================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
+    :target: https://github.com/OCA/manufacture/tree/14.0/mrp_attachment_mgmt
+    :alt: OCA/manufacture
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_attachment_mgmt
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/129/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to get all attachments of a bill of materials (all levels) on a view.
+This module allows to get all attachments of a work orders products to produce on a view.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Manufacturing -> Configuration -> Settings* and check "Work Orders" option.
+
+Usage
+=====
+
+#. Go to *Manufacturing -> Operation > Work Orders* and click in window open button (after the "Start" and "Block" options).
+#. The smart-button "Attachments" display the attachments of the related product.
+
+
+#. Go to *Manufacturing -> Products > Bill of material -> Desk Combination*.
+#. Go to "Attachment" icon related to "Office Chair Black" component and upload some file.
```

Comment:
I get this error:
```
int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
![Peek 08-03-2022 12-40](https://user-images.githubusercontent.com/5040182/157231590-d9944930-91ce-4c5a-bc47-8f883308abce.gif)

Reply:
Sorry, it is necessary to upload the files from the other icon and use the smart-button only to see all the attachments.
![ejemplo](https://user-images.githubusercontent.com/4117568/157279382-ff150b0d-4ec7-4721-bee0-ac1642da81d9.png)